distros/centos-stream/foreman-staging.repo

```diff
@@ -0,0 +1,33 @@
+[foreman-$foremanver-staging]
+name=Foreman $foremanver staging EL$releasever
+baseurl=https://stagingyum.theforeman.org/foreman/$foremanver/el$releasever/$basearch/
+module_hotfixes=1
```

Comment:
is this required given you actually enable modules?

Reply:
Maybe not, probably a leftover from earlier testing.